types/TagTypes.ts

```diff
@@ -30,3 +30,12 @@ export interface PixivAutoCompleteV2 {
     translated_name: string | null
   }[]
 }
+
+export interface PixivCandidates {
+  candidates: {
+    tag_name: string,
+    access_count: string,
+    tag_translation: string,
```

Comment:
Often pixiv api returns `null` when it doesn't exist. I doubt that every tag is translated, so unless you know it just returns an empty string I would append null types to `tag_translation`.

Reply:
Changed the `tag_translation` to optional property. Pixiv either returns `tag_translation` or nothing at all.
example
```
{tag_name: "loli", access_count: "762909547", type: "prefix"}
```